app/config/AppConfig.scala

```diff
@@ -28,6 +29,10 @@ class AppConfig @Inject()(
   @Named("appName") serviceIdentifier: String
 ) {
 
+  runModeConfiguration
+    .getOptional[String]("mongodb.uri")
+    .map(uri => MongockConfig(uri))
+
```

Comment:
Should we have some logic to handle missing configuration? I mean some `.getOrElse(throw Exception`?

Reply:
I did this deliberately as I didn't want to add "mongodb.uri" config to the tests.    If this config value is missing then we have bigger problems than not running the upgrade scripts.